og.module

```diff
@@ -230,6 +231,23 @@ function og_field_formatter_info_alter(array &$info) {
 }
 
 /**
+ * Implements hook_field_widget_info_alter().
+ */
+function og_field_widget_info_alter(array &$info) {
+  $info['options_buttons']['field_types'][] = OgGroupAudienceHelper::GROUP_REFERENCE;
+}
+
+/**
+ * Implements hook_field_widget_form_alter().
+ */
+function og_field_widget_form_alter(&$element, FormStateInterface $form_state, $context) {
+  $field_definition = $context['items']->getFieldDefinition();
+  if ($field_definition->getType() == OgGroupAudienceHelper::GROUP_REFERENCE && $element['#type'] == 'radios') {
```

Comment:
tests seem to [fail](https://travis-ci.org/amitaibu/og/jobs/157357023#L609) over this line